src/index.js

```diff
@@ -34,11 +34,13 @@ class DropdownTreeSelect extends Component {
     onFocus: PropTypes.func,
     onBlur: PropTypes.func,
     simpleSelect: PropTypes.bool,
+    radioSelect: PropTypes.bool,
     noMatchesText: PropTypes.string,
     showPartiallySelected: PropTypes.bool,
     disabled: PropTypes.bool,
     readOnly: PropTypes.bool,
-    hierarchical: PropTypes.bool
+    hierarchical: PropTypes.bool,
```

Comment:
We need to add trailing commas so that we can avoid noise in diffs like these. :)

Reply:
![bild](https://user-images.githubusercontent.com/17863113/54885785-490ed500-4e80-11e9-9c1b-7f173d2de0b1.png)
You have a rule enabled preventing that

Reply:
https://eslint.org/docs/rules/comma-dangle what should we target?

Reply:
https://github.com/airbnb/javascript/blob/b85baeafed8b66fdd9756439a0b8774860147913/packages/eslint-config-airbnb-base/rules/style.js

Goes with always-multiline for all types. Could simple be `comma-dangle: "always-multiline"` I guess.

Reply:
I would go with `trailing-commas: "es5"` https://prettier.io/docs/en/options.html#trailing-commas

I started playing with a prettierrc in a separate branch but I ran into some weird prettier plugin errors. 

I would also do it after we merge all 3 of your PRs otherwise it'd just create a lot of extra work for you to update from develop branch.